### PR TITLE
Add typings to 'just-has'

### PR DESCRIPTION
**Pull request by daniel0mullins**
This PR adds Typescript types to 'object-has' package

### CODE REVIEW

File: packages/object-has/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Definitions by: daniel0mullins <https://github.com/daniel0mullins>
+declare function has(obj: object | undefined, propsArg: Array<string> | Symbol | string): boolean;
+export default has;
```
Comment (angus-c) [inline packages/object-has/index.d.ts:3]: missing newline at end of file

File: packages/object-has/index.tests.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+has(undefined, 'a');
+const sym = Symbol('foo');
+has({[sym]: 1}, sym);
+
+// NOT OK
+
+// @ts-expect-error
+has()
+// @ts-expect-error
+has({}, () => {});
```
Comment (angus-c) [inline packages/object-has/index.tests.ts:16]: missing newline at end of file

File: packages/object-has/index.tests.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import has from './index';
+
+// OK
+
+has({a: 1}, 'a');
+has({a: {b: 1}}, 'a.b');
+has(undefined, 'a');
+const sym = Symbol('foo');
+has({[sym]: 1}, sym);
```
Comment (angus-c) [inline packages/object-has/index.tests.ts:9]: can you add a test with array of strings as second arg

File: packages/object-has/index.tests.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+// NOT OK
+
+// @ts-expect-error
+has()
```
Comment (angus-c) [inline packages/object-has/index.tests.ts:14]: can you add a not ok test with just one arg

**Review by angus-c — COMMENTED** [review]
Thanks this is great! Just a few nits.

File: packages/object-has/index.tests.ts (hunk 3 above)
Comment (daniel0mullins) [reply packages/object-has/index.tests.ts:9]: Added

**Comment by angus-c** [comment]
thank you!

**Comment by angus-c** [comment]
thank you!

**Pull request merged**